### PR TITLE
Added context_processors.py

### PR DESCRIPTION
**Pull request by davecranwell**
We always need to know DEBUG. BASE_URL i'm less certain of but always seems included.

### CODE REVIEW

**Comment by kaedroho** [comment]
``DEBUG`` should be available in ``django.template.context_processors.debug`` which is switched on by default.

**Comment by davecranwell** [comment]
Do you mean `{{ DEBUG }}` in my templates should automatically work without this? If not what's the template code?

File: {{cookiecutter.repo_name}}/.template-version [hunk 1]
```diff
@@ -1 +1 @@
-v2.3.1
+v2.3.2
```
Comment (kaedroho) [inline {{cookiecutter.repo_name}}/.template-version]: Version should be bumped post-merge

File: {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/context_processors.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import urllib
+
+from django.conf import settings
+
+def global_vars(request):
+    return {
+        'BASE_URL': getattr(settings, 'BASE_URL', ""),
+        'DEBUG': getattr(settings, 'DEBUG', ""),
+    }
```
Comment (kaedroho) [inline {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/context_processors.py]: Missing newline at end of file

File: {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/context_processors.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import urllib
+
+from django.conf import settings
+
```
Comment (kaedroho) [inline {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/context_processors.py]: Two lines needed between imports and function

**Comment by kaedroho** [comment]
@davecranwell 

It should be lowercase ``{{ debug }}``

**Comment by kaedroho** [comment]
I've never had to use the BASE_URL before. You should probably use ``{{ request.site.root_url  }}``

**Comment by davecranwell** [comment]
Indeed that's what I normally use too. I'm not sure why it gets added. And if `{{ debug }}` works already, this whole PR is redundant, I think.

File: {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/context_processors.py [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+def global_vars(request):
+    return {
+        'BASE_URL': getattr(settings, 'BASE_URL', ""),
+        'DEBUG': getattr(settings, 'DEBUG', ""),
```
Comment (kaedroho) [inline {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/context_processors.py]: DEBUG should be provided by Djangos builtin context ``debug`` context processor. Which is already enabled: https://github.com/torchbox/wagtail-cookiecutter/blob/master/%7B%7Bcookiecutter.repo_name%7D%7D/%7B%7Bcookiecutter.repo_name%7D%7D/settings/base.py#L79

``BASE_URL`` is only used for emails, you should use ``request.site.root_url`` instead.

**Pull request closed without merging**